src/pages/docs/tutorials/tutorial-kitten-browser/data-and-state/index.md

```diff
@@ -169,7 +169,7 @@ Instead, you'll most often apply these using the [rest and spread operators](../
 
 > **Advanced**
 >
-> In most cases, wrapping a component with Spottable is sufficient to make it navigable and selectable. However, in the case of Kitten, it won't be selectable because we've hijacked the `onClick` handler for our custom `onSelect` event. The current solution is to wrap the root element (`<div>`) with Spottable instead of the component (`Kitten`) but that is left as an exercise for the reader.
+> In most cases, wrapping a component with Spottable is sufficient to make it navigable and selectable. However, in the case of Kitten, it won't be selectable because we've hijacked the `onClick` handler for our custom `onSelect` event. The current solution is to wrap the root element (`<div>`) with Spottable instead of the component (`Kitten`) but that is left as an exercise for the reader. The answer is in the Enact samples [repository](https://github.com/enactjs/samples/blob/develop/sandstone/tutorial-kitten-browser/src/components/Kitten/Kitten.js).
```

Comment:
How about..

```suggestion
> In most cases, wrapping a component with Spottable is sufficient to make it navigable and selectable. However, in the case of Kitten, it won't be selectable because we've hijacked the `onClick` handler for our custom `onSelect` event. The current solution is to wrap the root element (`<div>`) with Spottable instead of the component (`Kitten`) but that is left as an exercise for the reader. If you would like to check the answer, you can find it [here](https://github.com/enactjs/samples/blob/develop/sandstone/tutorial-kitten-browser/src/components/Kitten/Kitten.js).
```